src/main/java/gift/validation/ForbiddenWordValidator.java

```diff
@@ -0,0 +1,20 @@
+package gift.validation;
+
+import jakarta.validation.ConstraintValidator;
+import jakarta.validation.ConstraintValidatorContext;
+
+public class ForbiddenWordValidator implements ConstraintValidator<ForbiddenWord, String> {
+
+    private String forbidden;
+
+    @Override
+    public void initialize(ForbiddenWord constraintAnnotation) {
+        this.forbidden = constraintAnnotation.word();
+    }
+
+    @Override
+    public boolean isValid(String value, ConstraintValidatorContext context) {
+        if (value == null) return true; // @NotBlank와 같이 사용해야 함
```

Comment:
추후 if 문에 로직이 추가되었을 때, 실수할 여지가 있기 때문에 가능한 `{ }` 블록으로 if문을 감싸서 코딩하는 습관을 들이시면 좋을 것 같아요. 😃 

```suggestion
        if (value == null) {
            return true;
        }
```